warehouse/upload.go

```diff
@@ -781,6 +783,19 @@ func (job *UploadJob) UpdateTableSchema(tName string, tableSchemaDiff warehouseu
 }
 
 func (job *UploadJob) alterColumnsToWarehouse(tName string, columnsMap model.TableSchema) error {
+	if !job.AlterEnabled {
+		pkgLogger.Debugw("skipping alter columns to warehouse",
+			logfield.SourceID, job.warehouse.Source.ID,
+			logfield.SourceType, job.warehouse.Source.SourceDefinition.Name,
+			logfield.DestinationID, job.warehouse.Destination.ID,
+			logfield.DestinationType, job.warehouse.Destination.DestinationDefinition.Name,
+			logfield.WorkspaceID, job.warehouse.WorkspaceID,
+			logfield.TableName, tName,
+			"columns", fmt.Sprint(columnsMap),
```

Comment:
`fmt.Sprintf` is probably not needed. just saying though, it's not required to remove it.